aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -142,48 +152,93 @@ HTTP.sys delegates to kernel mode authentication with the Kerberos authenticatio
+   ```
 
-   1. The built-in tool for configuring HTTP.sys is *netsh.exe*. *netsh.exe* is used to reserve URL prefixes and assign X.509 certificates. The tool requires administrator privileges.
+   * `<IP>` &ndash; Specifies the local IP address for the binding. Don't use a wildcard binding. Use a valid IP address.
+   * `<PORT>` &ndash; Specifies the port for the binding.
+   * `<THUMBPRINT>` &ndash; The X.509 certificate thumbprint.
+   * `<GUID>` &ndash; A developer-generated GUID to represent the app for informational purposes.
 
-      The following example shows the commands to reserve URL prefixes for ports 80 and 443:
+   In the following example:
 
-      ```console
-      netsh http add urlacl url=http://+:80/ user=Users
-      netsh http add urlacl url=https://+:443/ user=Users
-      ```
+   * The local IP address of the server is `10.0.0.4`.
+   * An online random GUID generator provides the `appid` value.
+
+   ```console
+   netsh http add sslcert 
+       ipport=10.0.0.4:443 
+       certhash=b66ee04419d4ee37464ab8785ff02449980eae10 
+       appid="{9412ee86-c21b-4eb8-bd89-f650fbf44931}"
+   ```
+
+   When a certificate is registered, the tool responds with `SSL Certificate successfully added`.
+
+   To delete a certificate registration, use the `delete sslcert` command:
+
+   ```console
+   netsh http delete sslcert ipport=<IP>:<PORT>
+   ```
 
-      The following example shows how to assign an X.509 certificate:
+   Reference documentation for *netsh.exe*:
 
-      ```console
-      netsh http add sslcert ipport=0.0.0.0:443 certhash=MyCertHash_Here appid="{00000000-0000-0000-0000-000000000000}"
-      ```
+   * [Netsh Commands for Hypertext Transfer Protocol (HTTP)](https://technet.microsoft.com/library/cc725882.aspx)
+   * [UrlPrefix Strings](https://msdn.microsoft.com/library/windows/desktop/aa364698.aspx)
 
-      Reference documentation for *netsh.exe*:
+1. Run the app.
 
-      * [Netsh Commands for Hypertext Transfer Protocol (HTTP)](https://technet.microsoft.com/library/cc725882.aspx)
-      * [UrlPrefix Strings](https://msdn.microsoft.com/library/windows/desktop/aa364698.aspx)
+   Administrator privileges aren't required to run the app when binding to localhost using HTTP (not HTTPS) with a port number greater than 1024. For other configurations (for example, using a local IP address or binding to port 443), run the app with administrator privileges.
 
-   2. Create self-signed X.509 certificates, if required.
+   The app responds at the server's public IP address.
 
-      [!INCLUDE [How to make an X.509 cert](~/includes/make-x509-cert.md)]
+   A development certificate is used in this example. The page loads securely after bypassing the browser's untrusted certificate warning.
 
-4. Open firewall ports to allow traffic to reach HTTP.sys. Use *netsh.exe* or [PowerShell cmdlets](https://technet.microsoft.com/library/jj554906).
+   ![Browser window showing the app's Index page loaded](httpsys/_static/browser.png)
```

Comment:
This image is confusing as the address bar gives one IP and the page prints another. If you're going to keep this then mention they don't match because you were using a cloud hosted VM where the public IP is different from the local one.

Reply:
Ah, yes, I forgot/missed adding the public IP to the line leading into this ...

> The app responds at the server's public IP address.

I'll enhance this on the next commit.